pkg/util/proto/openapi.go

```diff
@@ -242,6 +242,21 @@ func (p *Primitive) GetName() string {
 	return fmt.Sprintf("%s (%s)", p.Type, p.Format)
 }
 
+// AnyValue is an arbitrary value (object, array or primitive)
```

Comment:
How is one supposed to use that object? It doesn't have any subtype? any data? What abstraction is it mapping to? Can you point me to the specific OpenAPI specification section describing "any"?

Reply:
We'll also probably need to change the SchemaVisitor to add that.